### PR TITLE
add support for compressed kernel modules with xz

### PR DESCRIPTION
**Pull request by tpgxyz**
Hi,

this pull request add support for compressed kernel modules with xz (ko.tar.xz)

Original sources are from OpenMandriva.

### CODE REVIEW

**Comment by jared-dominguez** [comment]
@tpgxyz do you have any responses to @scaronni's feedback?

**Comment by tpgxyz** [comment]
@jared-dominguez  @scaronni Guys i'm busy now, and then going on hollidays for couple of weeks. I'll push a fixes when i'll have time. Please be patient.

**Comment by scaronni** [comment]
Sure, no problem. Thanks.

**Comment by scaronni** [comment]
No activity, closing down.
Would be nice to have though.

File: dkms [hunk 1]
```diff
@@ -169,6 +169,14 @@ set_module_suffix()
     kernel_test="${1-:$(uname -r)}"
     module_suffix=".ko"
     [[ $(VER $kernel_test) < $(VER 2.5) ]] && module_suffix=".o"
+    grep -q '\.gz:' /lib/modules/$kernel_test/modules.dep 2>/dev/null && gz_modules=1
```
Comment (atar-axis) [inline dkms]: Why not checking the kernel config for MODULE_COMPRESS_XZ / MODULE_COMPRESS_GZIP ?

**Pull request closed without merging**